style.css

```diff
@@ -0,0 +1,11 @@
+header{
+    position: fixed;
+    width: 100%;
+    top: 0;
+    left: 0;
+}
```

Comment:
Please adjust the positioning of the closing brace of your selector for cleaner code.